interpreter/llvm/src/cmake/config-ix.cmake

```diff
@@ -372,6 +372,8 @@ elseif (LLVM_NATIVE_ARCH MATCHES "hexagon")
   set(LLVM_NATIVE_ARCH Hexagon)
 elseif (LLVM_NATIVE_ARCH MATCHES "s390x")
   set(LLVM_NATIVE_ARCH SystemZ)
+elseif (LLVM_NATIVE_ARCH MATCHES "s390")
```

Comment:
The pr #153 was written in order to make the cmake build do the same as the configure build, and at the time the configure build did have support for s390. This was the reason for the proposed changes here - not to loose functionality when switching to the cmake build. But you are right - llvm does not support s390. So a better way to remove the differences between configure and cmake would have been to drop the s390 support from the configure build rather than adding it to the cmake build. I was not aware of this detail at the time.

Just make sure that support for s390x (which is supported by llvm) is not lost when you remove the s390 support. I don't know if it still is the case, but it used to be the case that in same places in the code tree the substring match on "s390" was supposed to match both "s390" and "s390x".